.github/workflows/linux_riscv.yml

```diff
@@ -61,9 +61,11 @@ jobs:
       OPENVINO_BUILD_DIR: /__w/openvino/openvino/openvino_build
       INSTALL_DIR: /__w/openvino/openvino/openvino_install
       CONAN_USER_HOME: /mount/caches/ccache/ubuntu22_riscv64_master_release/.conan
-      CCACHE_DIR: /mount/caches/ccache/ubuntu22_riscv64_master_release
+      CCACHE_REMOTE_DIR: /mount/caches/ccache/ubuntu22_riscv64
+      CCACHE_DIR: /__w/openvino/openvino/ccache
       CCACHE_TEMPDIR: /__w/openvino/openvino/ccache_temp
-      CCACHE_MAXSIZE: 50G
+      CCACHE_MAXSIZE: 5G
```

Comment:
Why reduce the cache size? Isn't it like >1TB of storage?

Reply:
this is ccache size, not the size of the share where it will be saved as archive